Mathlib/Order/Basic.lean

```diff
@@ -882,7 +882,8 @@ end Preorder
 /-- The pointwise partial order on a product.
     (The lexicographic ordering is defined in `Order.Lexicographic`, and the instances are
     available via the type synonym `α ×ₗ β = α × β`.) -/
-instance (α : Type u) (β : Type v) [PartialOrder α] [PartialOrder β] : PartialOrder (α × β) where
+instance partialOrder (α : Type u) (β : Type v) [PartialOrder α] [PartialOrder β] :
```

Comment:
What is the reason to force lowercase `p` here?

Reply:
It was my understanding that instances are named like defs with lowerCamelCase, is that wrong? Alternatively I could try to drop the explicit names

Reply:
I don't know. I started looking at mathlib 4 port less than 24h ago. So, I'm asking stupid questions. It seems strange to me to use explicit names for all instances.

Reply:
I removed the explicit names for everything except
```
OrderDual.semilatticeSup
OrderDual.semilatticeInf
OrderDual.distribLattice
```

Reply:
*If* you wanted to name this instance, this would be the correct name. The rule is that CamelCase names get turned into lowerCamelCase when used in definitions referring to them.